src/settings/KnowledgeBaseListView.ts

```diff
@@ -50,8 +50,13 @@ export class KnowledgeBaseListView implements UpdatableSettingsViewer {
 		const listConfig: ListConfig<KnowledgeBaseEntry, KnowledgeBaseRow> = {
 			rowHeight: size.list_row_height,
 			fetch: async (startId, count) => {
-				const items = await this._entityClient.loadRange(KnowledgeBaseEntryTypeRef, knowledgebaseListId, startId, count, true)
-				return { items, complete: items.length < count }
+				if (startId === GENERATED_MAX_ID) {
+					// load all entries at once to apply custom sort order
+					const allEntries = await this._entityClient.loadAll(KnowledgeBaseEntryTypeRef, knowledgebaseListId)
+					return { items: allEntries, complete: true }
+				} else {
+					throw new Error("fetch contact called for specific start id")
```

Comment:
that's probably copied from contactlistview? the text needs to be adjusted and I would mention that it's copied in both places

Reply:
yea I forgot to change that. Were would you mention that? In a comment?

Reply:
yep, comment is fine